include/experimental/__p1673_bits/blas1_matrix_frob_norm.hpp

```diff
@@ -51,10 +51,15 @@ namespace experimental {
 inline namespace __p1673_version_0 {
 namespace linalg {
 
-template<class in_matrix_t,
-         class Scalar>
-Scalar matrix_frob_norm(in_matrix_t A,
-                        Scalar init)
+template<
+    class ElementType,
+    class Extents,
```

Comment:
Instead of an `Extents` template parameter, would you consider instead adding `ptrdiff_t numRows` and `ptrdiff_t numCols` template parameters, and using `std::experimental::extents<numRows, numCols>` instead of `Extents` in the type of `A`?

Reply:
This would make the matrix functions more consistent with the vector functions below.

Reply:
Will do!